--- a/src/fixed/common.h
+++ b/src/fixed/common.h
@@ -1099,6 +1099,10 @@ struct Texture
 #ifdef __3DO__
     uint8* data;
     uint32 shift;
+#elif defined(HOST_BUILD)
+    uintptr_t tile;
+    uint32 uv01;
+    uint32 uv23;
 #else
     uint32 tile;
     uint32 uv01;
@@ -1111,6 +1115,10 @@ struct Sprite
 #ifdef __3DO__
     uint32 texture;
     int32 l, t, r, b;
+#elif defined(HOST_BUILD)
+    uintptr_t tile;
+    uint32 uwvh;
+    int16 l, t, r, b;
 #else
     uint32 tile;
     uint32 uwvh;
